meta-mender-qemu/recipes-core/systemd/files/01-logs-to-console.conf

```diff
@@ -0,0 +1,2 @@
+[Journal]
+ForwardToConsole=true
```

Comment:
Here: https://asciinema.org/a/71qu9JDI21yqOapq1INsY2Duz

Now, because the state changes are logged directly to console, they will also appear in docker logs during integration tests (super useful). For acceptance, the tests that use qemu disable mender agent service, so no logs appear.

The only thing that slightly annoys me is that journald will replay dmesg when it starts. I could not find a way to stop this behavior.

I would propose that we keep this as a default and have an env switch that can be applied when you run a docker container. For local development you can just comment out the whole append or change `ForwardToConsole` to false.